### PR TITLE
add fdint recipe

### PR DESCRIPTION
**Pull request by basnijholt**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by basnijholt** [comment]
@conda-forge/help-python, I don't understand how this could possibly lead to unresolvable dependencies...

It literally only requires `numpy`.
```
conda.exceptions.UnsatisfiableError: The following specifications were found to be incompatible with each other

Output in format: Requested package -> Available versions

Package _libgcc_mutex conflicts for:
python=3.8[build=*_cpython] -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1',build='main|conda_forge']
numpy=1.9 -> libgcc-ng[version='>=7.3.0'] -> _libgcc_mutex[version='*|0.1',build='main|conda_forge']

Package python_abi conflicts for:
pip -> python[version='>=3'] -> python_abi[version='3.6.*|3.6|3.8.*|3.7.*',build='*_cp38|*_pypy36_pp73|*_cp37m|*_cp36m']
numpy=1.9 -> python[version='>=3.6,<3.7.0a0'] -> python_abi==3.6[build=*_pypy36_pp73]

Package python conflicts for:
numpy=1.9 -> numpy-base[version='>=1.9.3,<2.0a0'] -> python[version='>=3.8,<3.9.0a0']
pip -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3|>=3.8,<3.9.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0|3.4.*']
numpy=1.9 -> python[version='2.7.*|3.4.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0']
python=3.8[build=*_cpython]
Note that strict channel priority may have removed packages required for satisfiability.
```

**Comment by scopatz** [comment]
Yeah that is pretty weird. I tried restarting

**Comment by basnijholt** [comment]
Unfortunately, it didn't work.

I will try rebasing...

File: recipes/fdint/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  imports:
+    - fdint
+    - fdint.examples
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://scott-maddox.github.io/fdint
```
Comment (scopatz) [inline recipes/fdint/meta.yaml:38]: This site does not seem to exist

Comment (basnijholt) [reply recipes/fdint/meta.yaml:38]: Good catch! I had already fixed this, but it must have gotten lost in the rebase/force pushing...

File: recipes/fdint/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - numpy
+    - pip
+    - python
```
Comment (scopatz) [inline recipes/fdint/meta.yaml:23]: You probably need to add `cython` here

**Comment by basnijholt** [comment]
I am still seeing:
```
Output in format: Requested package -> Available versions

Package python conflicts for:
pip -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3|>=3.8,<3.9.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0|3.4.*']
numpy=1.9 -> python[version='2.7.*|3.4.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0']
cython -> python_abi=3.8[build=*_cp38] -> python[version='3.7.*|3.8.*']
numpy=1.9 -> numpy-base[version='>=1.9.3,<2.0a0'] -> python[version='>=3.8,<3.9.0a0']
cython -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0|3.4.*']
python=3.8[build=*_cpython]

Package _openmp_mutex conflicts for:
cython -> libgcc-ng[version='>=7.5.0'] -> _openmp_mutex[version='>=4.5']
numpy=1.9 -> libgcc-ng[version='>=7.3.0'] -> _openmp_mutex[version='*|>=4.5',build=*_llvm]
python=3.8[build=*_cpython] -> libgcc-ng[version='>=7.5.0'] -> _openmp_mutex[version='>=4.5']

Package bzip2 conflicts for:
numpy=1.9 -> python[version='>=2.7,<2.8.0a0'] -> bzip2[version='>=1.0.6,<2.0a0|>=1.0.8,<2.0a0']
pip -> python[version='>=3'] -> bzip2[version='>=1.0.6,<2.0a0|>=1.0.8,<2.0a0']
cython -> python[version='>=3.6,<3.7.0a0'] -> bzip2[version='>=1.0.6,<2.0a0|>=1.0.8,<2.0a0']

Package certifi conflicts for:
pip -> setuptools -> certifi[version='>=2016.09|>=2016.9.26']
cython -> setuptools -> certifi[version='>=2016.09|>=2016.9.26']
Note that strict channel priority may have removed packages required for satisfiability.
```

I am noticing this in https://github.com/conda-forge/staged-recipes/pull/12116 too. Not sure why?

**Comment by scopatz** [comment]
CC @conda-forge/staged-recipes

**Pull request closed without merging**